### PR TITLE
readme: add note to reinstall rustfmt after toolchain update

### PR DESCRIPTION
**Pull request by brotzeit**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -136,6 +136,8 @@ more Emacs-y.
    [Rust installed](https://www.rust-lang.org/en-US/install.html). 
    The file `rust-toolchain` indicates the version that gets installed.
    This happens automatically, so don't override the toolchain manually.
+   IMPORTANT: When the toolchain was updated, you have to reinstall
```
Comment (shaleh) [inline README.md]:
```suggestion
   IMPORTANT: Whenever the toolchain updates, you have to reinstall
```

File: README.md [hunk 2]
```diff
@@ -136,6 +136,8 @@ more Emacs-y.
...
+   rustfmt manually. PRs to fix this issue are very welcome.
```
Comment (shaleh) [inline README.md]: There is no change we can make. This is a failure in `rustup`.

Comment (brotzeit) [reply README.md]: I thought this maybe could be bypassed somehow.

Comment (shaleh) [reply README.md]: I suspect the -preview items are not updated automatically. So as long as we use nightly and rustfmt/clippy stay preview this is way it is.

**Pull request merged**